teacher.rb

```diff
@@ -0,0 +1,10 @@
+require './person'
+class Teacher < Person
+  def initialize(_spcialization)
+    super(name, age, parent_permission)
+  end
```

Comment:
- [ ] Same thing here, kindly assign the others in the initialize. Also, you probably forgot to assign the specialization, it would be something like `@specialization = _specialization`